public/js/angular/controllers.js

```diff
@@ -603,10 +603,19 @@ angular
       };
     }
   ])
-  .controller('gogglesController', ['$scope', 'CONFIG',
-    function ($scope, config) {
-      $scope.gogglesUrl = config.gogglesUrl;
+  .controller('gogglesController', ['$scope', 'CONFIG', 'wmAnalytics',
+    function ($scope, config, analytics) {
 
+      if ($scope.$parent.currentPath.indexOf('/install') === -1) {
+        //User is on goggles landing page
+        analytics.event('Goggles landing page');
+      }
+      else {
+        //User is on goggles install page
+        analytics.event('Goggles install page');
+      }
```

Comment:
We don't need this section (609-616), because we'll already have pageview tracking from the basic GA tags on the page.